racketscript-compiler/racketscript/compiler/runtime/kernel.js

```
@@ -1,58 +1,65 @@
 import * as Core from "./core.js";
 
+// This is only here temporary until we fix $/typeof.
```

Let the test fail since its not this code problem, but of `$/typeof`. As long as we have an issue/test to track this bug its better to not fix it here. If something else unrelated to patch breaks just make sure there exist an issue/test to track that. You can add a note in commit message on what what breaks. The reason is simply good history and easy reverts in rare cases.

---

Unfortunately not doing this makes a number of other tests fails with no clear cause.
I have opened an issue #81, and will open another one specifically about `$/typeof` once this is merged though